Dockerfile

```diff
@@ -2,6 +2,8 @@ from alerta/alerta-web
 
 LABEL maintainer="Artem Goncharov <artem.goncharov@gmail.com>"
 
+RUN /venv/bin/pip install requests --upgrade
```

Comment:
since we dropped dependency that was requiring it - can be also dropped now

Reply:
dropped